File: apps/Applications/ios/Local For Vocal/Local For Vocal/Sources/Utils/AppTheme.swift
import SwiftUI

struct AppTheme {
    struct Colors {
        static let primary = Color(hex: "#2563EB")
        static let secondary = Color(hex: "#10B981")  // Or #1F2937 depending on usage
        static let background = Color(hex: "#F3F4F6")

        static let textPrimary = Color(hex: "#1F2937")
        static let textSecondary = Color(hex: "#4B5563")  // Adjusted to match UI usage
        static let textTertiary = Color(hex: "#6B7280")

        static let error = Color(hex: "#EF4444")
        static let warning = Color(hex: "#F59E0B")
        static let success = Color(hex: "#16A34A")

        static let border = Color(hex: "#E5E7EB")
        static let backgroundLight = Color(hex: "#F9FAFB")
        static let backgroundWhite = Color.white
    }

    struct Constants {
        static let cornerRadius: CGFloat = 8
        static let padding: CGFloat = 16
        static let animationDuration: Double = 0.3
    }
}

File: apps/Applications/android/app/src/main/java/com/localforvocalstartup/app/utils/AppTheme.kt
package com.localforvocalstartup.app.utils

import androidx.compose.ui.graphics.Color
import androidx.compose.ui.unit.dp

object AppTheme {
    object Colors {
        val primary = Color.fromHex("#2563EB")
        val secondary = Color.fromHex("#10B981")
        val background = Color.fromHex("#F3F4F6")

        val textPrimary = Color.fromHex("#1F2937")
        val textSecondary = Color.fromHex("#4B5563")
        val textTertiary = Color.fromHex("#6B7280")

        val error = Color.fromHex("#EF4444")
        val warning = Color.fromHex("#F59E0B")
        val success = Color.fromHex("#16A34A")

        val border = Color.fromHex("#E5E7EB")
        val backgroundLight = Color.fromHex("#F9FAFB")
        val backgroundWhite = Color.White
    }

    object Constants {
        val cornerRadius = 8.dp
        val padding = 16.dp
        const val animationDuration = 300 // ms mapped from Double 0.3s
    }
}
